### DISCUSSION TITLE
"Validation On Blur" example: bug or feature?

### DISCUSSION THREAD

**Discussion by roman-petrov** [react-hook-form/react-hook-form#4158] [Q&A] [answered]
There is [Validation On Blur](https://codesandbox.io/s/w7p3km6nyw) example at https://github.com/react-hook-form/react-hook-form/tree/master/examples.

What seems to be correct: I do not touch any input and click "Submit"  -> all fields marked as invalid.
What seems to be incorrect: I click on first field and click "Submit" -> only first field marked as invalid.

I discovered this case in our applications so I wonder is it a bug or not?

**Comment by bluebill1049** [comment, marked as answer]
<img width="778" alt="Screen Shot 2021-02-11 at 8 09 49 pm" src="https://user-images.githubusercontent.com/10513364/107618092-1a34be80-6ca5-11eb-8a27-81508c8ffcba.png">

> What seems to be incorrect: I click on first field and click "Submit" -> only first field marked as invalid.

I see all inputs errors above

**Reply by roman-petrov** [reply]
Саn you please follow these steps (I tested in both latest Chrome / FF):
- Click on first input field and (optionally) type something
- Click on "Submit"

I recorded a video demonstrating this https://gyazo.com/8f94057b72182003ae63880f7c1e3e35. It seems the issue is a bit inconsistent...

**Reply by bluebill1049** [reply]
because the submit button position got pushed, if you make an error message fixed height, I reckon it would fix the problem.

**Reply by roman-petrov** [reply]
I am sorry but I don't get you... So is the example working correctly or there is a bug in example or `react-hook-form` library?..

**Reply by bluebill1049** [reply]
https://codesandbox.io/s/w7p3km6nyw?file=/src/index.js take a look at this exmaple.

**Reply by roman-petrov** [reply]
Thank you! We will try to make the same fix in our application.

**Reply by bluebill1049** [reply]
Just give a fixed height for the error message, so the button doesn't "Jump"

**Reply by roman-petrov** [reply]
great, thank you very much!
